### PR TITLE
responsive

### PR DESCRIPTION
**Pull request by ioandodita069**

### CODE REVIEW

File: html/responsive.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+        @media only screen and (max-width:500px) {
+
+            /*for mobile phones:*/
+            .menu,
+            .main,
+            .right {
+                width: 100%;
+            }
+        }
```
Comment (pancudaniel7) [inline html/responsive.html:58]: This setup looks nice, also please sent me 3 printscreens on this 3 different resolutions, default, 800px and 500px in the comment to see how it looks like.

File: html/responsive.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    </style>
+</head>
+
+<body style="font-family:Verdana;">
+
+    <div style="background-color:#dddddd;padding:15px;">
```
Comment (pancudaniel7) [inline html/responsive.html:64]: We talk about hits,  the Romanian law says  clearly that Ioan Dodita is not allowed to use **style** in **tags**. Please **respect** **this**!

File: html/responsive.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+        <div class="main">
+            <h2>The Walk</h2>
+            <p>The walk on the Greece island will take you about two and a half hours, but the view it is goreous.</p>
+            <img src="../img/greece.jpg" style="width:100%">
+        </div>
+
+        <picture>
+            <source srcset="../img/img2.jpg" media="(max-width:600px)" style=width:100%>
+            <source srcset="../img/img1.jpg" media="(max-widht:900px)" style=width:100%>
```
Comment (pancudaniel7) [inline html/responsive.html:85]: 👍🏻

Comment (ioandodita069) [reply html/responsive.html:111]: image

Comment (ioandodita069) [reply html/responsive.html:111]: image

File: html/responsive.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+            <div class="menuitem">The Walk</div>
+            <div class="menuitem">Transport</div>
+            <div class="menuitem">History</div>
+            <div class="menuitem">Gallery</div>
+        </div>
+
+        <div class="main">
+            <h2>The Walk</h2>
+            <p>The walk on the Greece island will take you about two and a half hours, but the view it is goreous.</p>
+            <img src="../img/greece.jpg" style="width:100%">
```
Comment (pancudaniel7) [inline html/responsive.html:106]: Remove this and add it to css

File: html/responsive.html [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+        </div>
+
+        <picture>
+            <source srcset="../img/img2.jpg" media="(max-width:600px)" style=width:100%>
+            <source srcset="../img/img1.jpg" media="(max-widht:900px)" style=width:100%>
```
Comment (pancudaniel7) [inline html/responsive.html:111]: You did not removed style here, remove all the style atributes from all the elements and add them to style section in head, the site should look the same

**Review by pancudaniel7 — COMMENTED** [review]
Please read again the new comments you did not understand what modifications you have to do

**Pull request merged**